cmd/ocsp-updater/main.go

```diff
@@ -479,12 +485,9 @@ func (updater *OCSPUpdater) missingReceiptsTick(ctx context.Context, batchSize i
 			updater.log.AuditErr(fmt.Errorf("Failed to get certificate: %s", err))
 			continue
 		}
-		// TODO(#1679) only submit to the logs we don't have a SCT for
-		err = updater.pubc.SubmitToCT(ctx, cert.DER)
-		if err != nil {
-			updater.log.AuditErr(fmt.Errorf("Failed to submit certificate to CT log: %s", err))
-			continue
-		}
+		ctx, cancel := context.WithTimeout(context.Background(), updater.grpcTimeout)
+		_ = updater.pubc.SubmitToCT(ctx, cert.DER)
+		cancel() // since we are in a for loop do this now instead of collecting a bunch
```

Comment:
// Cancel now instead of deferring because we are inn a loop

Also, should probably do something with the error